llama.cpp

```diff
@@ -2739,7 +2740,7 @@ size_t llama_save_session_file(struct llama_context * ctx, const char * path_ses
     llama_copy_state_data(ctx, state_data.get());
 
     file.write_u32('ggsn'); // magic
-    file.write_u32(0); // version
+    file.write_u32(1); // version
```

Comment:
nit: it's harmless but I don't believe it's necessary here to change the version? The binary format isn't affected by the decision of how many tokens to store.